aws/http_client.go

```diff
@@ -38,8 +38,9 @@ type HTTPClient interface {
 // of the BuildableHTTPClient. Only exported member values of the Transport and
 // optional Dialer will be copied between copies of BuildableHTTPClient.
 type BuildableHTTPClient struct {
-	transport *http.Transport
-	dialer    *net.Dialer
+	transport     *http.Transport
+	dialer        *net.Dialer
+	checkRedirect func(req *http.Request, via []*http.Request) error
```

Comment:
Is the `checkRedirect` behavior still used?

Reply:
Nope not needed, just missed cleaning up this member.